openstack/networking/v2/extensions/lbaas_v2/l7policies/testing/fixtures.go

```diff
@@ -0,0 +1,78 @@
+package testing
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/lbaas_v2/l7policies"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	"github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+// SingleL7PolicyBody is the canned body of a Get request on an existing l7policy.
+const SingleL7PolicyBody = `
+{
+	"l7policy": {
```

Comment:
It looks like this test fixture was copied from the API reference:

https://developer.openstack.org/api-ref/load-balancer/v2/index.html#create-an-l7-policy

We prefer that all test fixtures are generated in a working OpenStack environment. We've found too many cases where the API docs' fixtures do not match what OpenStack/LBaaS really responds with.

Reply:
I installed neutron-lbaas lib in my octavia devstack environment, the sample has been changed according to my test, just keep the uuids that are referenced in all the following PRs.